src/main/java/uk/gov/beis/els/categories/airconditioners/model/CoolingDuctlessAirConditionersForm.java

```diff
@@ -1,38 +1,48 @@
 package uk.gov.beis.els.categories.airconditioners.model;
 
+import io.swagger.v3.oas.annotations.media.Schema;
 import javax.validation.constraints.Digits;
 import javax.validation.constraints.NotBlank;
 import javax.validation.groups.Default;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.categories.airconditioners.service.AirConditionersService;
 import uk.gov.beis.els.categories.common.StandardTemplateForm50Char;
 import uk.gov.beis.els.categories.internetlabelling.model.InternetLabellingGroup;
 import uk.gov.beis.els.model.meta.DualModeField;
 import uk.gov.beis.els.model.meta.FieldPrompt;
 
+@Schema(name = "Cooling-only ductless air conditioner energy label")
 public class CoolingDuctlessAirConditionersForm extends StandardTemplateForm50Char {
 
   @FieldPrompt("Energy efficiency class for cooling")
   @NotBlank(message = "Select an energy efficiency indicator for cooling", groups = {Default.class, InternetLabellingGroup.class})
   @DualModeField
+  @ApiValuesFromLegislationCategory(serviceClass = AirConditionersService.class)
   private String coolingEfficiencyRating;
```

Comment:
might be worth adding `@NotBlank` to all required fields so they are marked as required in the API documentation
![image](https://user-images.githubusercontent.com/98316465/153190290-15bfc4bd-9a03-442c-87f9-7fa4f12a9a3d.png)

Reply:
The validation is already handled by the @Digits annotations, and so on. Adding @NotBlank breaks the form in those places.

Reply:
@FernandoRamirez-De-Aguilar You can mark fields as required by setting `required = true` in the `@Schema` annotation against a field, rather than setting `@NotBlank` or `@NotNull`

Reply:
@FernandoRamirez-De-Aguilar Ignore this. Use `@NotNull`